isyntax2raw/__init__.py

```diff
@@ -327,12 +327,17 @@ def acquisition_datetime(self):
             timestamp = str(pe_in.DICOM_ACQUISITION_DATETIME).strip()
         else:
             timestamp = pe_in.acquisition_datetime.strip()
-        # older files store the date time in YYYYmmddHHMMSS.ffffff format
+        # older files store the date time in YYYYmmddHHMMSS.ffffff format,
```

Comment:
If additional variants of the timestamp format start appearing in the wild, we might move towards something alongside the line of

```
for format in DATE_FORMATS:
    try:
        return datetime.strptime(timestamp, format)
    except ValueError:
        pass
 return parse(timestamp)
```